tests/FakeItEasy.Tests/Configuration/AnyCallConfigurationTests.cs

```diff
@@ -50,6 +50,18 @@ public void WithReturnType_should_set_the_type_to_the_configured_rule()
         }
 
         [Test]
+        public void WithNonVoidReturnType_should_set_the_flag_to_apply_to_all_return_types()
+        {
+            // Arrange
+
+            // Act
+            this.configuration.WithNonVoidReturnType();
+
+            // Assert
+            Assert.That(this.callRule.ApplicableToAllNonVoidReturnTypes, Is.EqualTo(true));

```

Comment:
I'm fine with it remaining. I'd care more if the type was public, but since it's internal.. meh.